README.md

```diff
@@ -1,2 +1,31 @@
 # Hello_Microverse
 A repo created to complete the "Hello Microverse" project.
+
+Hello Microverse Project
+In this project, I set up a "Hello world" repository. No complex coding was required for this exercise. My goal here was to master all of the tools and best practices I learned about in previous steps. As I will be using them in all Microverse projects and most likely in my future job as well, so it is important to know them!
+
+Built With
+HTML & CSS
+Frameworks
+
+
+Authors
+👤 Demix007
+
+GitHub: @demix007
+Twitter: @dat_dope_demix
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] Kindly be noticed that your README file should have the same fonts size and style as the Microverse [README template](https://github.com/microverseinc/readme-template/blob/master/README.md).